utils/buildJsonResults.js

```diff
@@ -4,6 +4,39 @@ const stripAnsi = require('strip-ansi');
 const constants = require('../constants/index');
 const path = require('path');
 
+// Borrowed from here: https://gist.github.com/john-doherty/b9195065884cdbfd2017a4756e6409cc/
```

Comment:
I would much prefer not doing direct string manipulation. It appears that libxmljs might be able to handle this - could you see if that's possible?

See: https://github.com/libxmljs/libxmljs/blob/master/src/xml_document.cc#L325

Reply:
The encoding lib is probably where this should go. 

Yeah, I was arguing with myself on where this should go. Now that you mention is, it should live in https://github.com/dylang/node-xml. No need for this library to know which characters are valid and which aren't. I'll close this PR.